types.d.ts

```diff
@@ -4,6 +4,13 @@ declare module 'lmnt-node' {
   /** The binary audio data provided by the server. */
   export type AudioData = Buffer;
 
+  /** The response from the server after synthesizing text. */
+  export type SynthesisResponse = {
+    audio: AudioData;
+    seed?: number;
+    durations?: Array<Object>;
```

Comment:
Can we provide a more detailed type for this? I think there's a way to specify the object details, etc.